backend/src/main/java/com/htecgroup/skynest/exception/accesstype/AccessTypeException.java

```diff
@@ -0,0 +1,13 @@
+package com.htecgroup.skynest.exception.accesstype;
+
+import com.htecgroup.skynest.exception.SkyNestBaseException;
+import org.springframework.http.HttpStatus;
+
+public class AccessTypeException extends SkyNestBaseException {
+
+  private static final long serialVersionUID = -6920883293393177324L;
```

Comment:
Whats the deal with all this serial ids inside all exceptions ?

Reply:
Throwable is Serializable

our exceptions extend Throwable 😔

Reply:
We didn't have one in the previous exceptions, thats why I'm asking.